packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/parser-struggles/main.js

```diff
@@ -10,6 +10,7 @@ function nothing() {
 
 // TODO: fix the export - .default should not be here
 function notMuch(item) {
+  //  throw Error(JSON.stringify(require('./is-ok')))
```

Comment:
debug line?

Reply:
yup. the fastest way to get output from code running in tests in endo :D